src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -1212,6 +1214,11 @@ public override void StatusEventHandler(object sender, BuildStatusEventArgs e)
                     propertyOutputMap[evaluationKey] = value;
                 }
             }
+            else if (e is BuildCanceledEventArgs buildCanceled && !_cancellationMessageRendered)
+            {
+                _cancellationMessageRendered = true;
+                Console.WriteLine(e.Message);
```

Comment:
This looks exactly the same as it did before, right? Just with a potential slight delay?

Reply:
Yes, it looks like before but there could be a slight delay.